packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -32,8 +32,8 @@
     1
   );
   $input-border-alpha: -0;
-  $color-input-disabled: color("disabled");
-  $color-input-disabled--weak: color("disabled-lighter");
+  $tile-border-color: color("gray-20");
```

Comment:
Any thoughts on making the border a bit darker at `gray-30`? 

Adds an extra step between the actual disabled element element and the default which might help to distinguish the difference between the two for lower visioned users

![Disabled comparison](https://github.com/uswds/uswds/assets/25211650/cb5155d2-e1b1-4a65-ae01-37adc0cc491d)

Reply:
I lean towards keeping the border color "gray-20". My reasoning is that I don't think that the border color should do much work to indicate that the element is disabled. Instead, it should be the lightened text and radio button color that carries that lift. 

In my mind, this PR primarily exists to restore the styles from 3.4.0, which are more closely aligned with "gray-20". Secondarily, the aim is to reduce visual noise to make this easier to read for users with visual processing disorders. Lightening the borders is likely not enough for that effort, but it is a step in the right direction. 

Curious what you and others think!

Reply:
That's totally fair! Definitely want to make sure it's readable for all. If that means relying more so on the text and radio/checkbox input color, so be it!